Start Local Db.ps1

```diff
@@ -0,0 +1,65 @@
+﻿param (
```

Comment:
Could we rename the file to Create Local Db?

Reply:
👍